debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/MethodMetaData.java

```diff
@@ -906,7 +906,7 @@ private static TR_ByteCodeInfoPointer getByteCodeInfo(VoidPointer inlinedCallSit
 		
 		public VoidPointer getInlinedMethod(VoidPointer inlinedCallSite) throws CorruptDataException
 		{
-			return TR_InlinedCallSitePointer.cast(inlinedCallSite)._methodInfo();
+			return VoidPointer.cast(TR_InlinedCallSitePointer.cast(inlinedCallSite)._method());
 		}
```

Comment:
@keithc-ca could you verify if the change to this java file is good?

Reply:
No, this is not good. See https://github.com/eclipse/omr/pull/5850#pullrequestreview-610186162.

Reply:
If the field name must be changed from `_methodInfo` to `_method`, then we'll need some (ugly) hacks to retain the ability to access core files that predate that name change.

Reply:
As this change has been reverted, I'll defer to those with JIT expertise.

Reply:
Thanks for your help with this Keith; would not have been good if something subtle like this slipped through.